docs/standard/serialization/system-text-json-configure-options.md

```diff
@@ -22,6 +25,7 @@ If you use `JsonSerializerOptions` repeatedly with the same options, don't creat
 The following code demonstrates the performance penalty for using new options instances.
 
 :::code language="csharp" source="snippets/system-text-json-configure-options/csharp/ReuseOptionsInstances.cs":::
+<!--code language="vb" source="snippets/system-text-json-configure-options/vb/ReuseOptionsInstances.vb"-->
```

Comment:
Why is this commented out?

Reply:
I didn't create a VB version for this, as its only purpose is to demonstrate the performance penalty for not reusing options instances.  I'll remove the comment.